src/main/java/lz/modelTesting/configuration/MongoDBConfig.java

```diff
@@ -35,11 +35,31 @@ CommandLineRunner loadSamples(UsersRepository usersRepository, MockupsRepository
                 usersRepository.save(new User("First 2 ", "Last 2", "u2@mail.pl", "$2y$12$xswpjLRaupJE8y4CHHIbjeba9BvlV.VLfut8q1Gqj5U2JDHgjNmuG"));
             }
 
+            Project[] projects = {new Project("First Project"), new Project("Sample Project"), new Project("Axure Project"), new Project("AdobeXD Project"), new Project("Figma Project")};
+
+            String basicType = "basic";
+            String surveyBody = "{\"questions\":[{\"isRequired\":false,\"enableIf\":true,\"name\":\"Text question?\",\"type\":\"comment\",\"title\":\"Text question?\"},{\"isRequired\":false,\"enableIf\":true,\"name\":\"Radiogroup question\",\"type\":\"radiogroup\",\"title\":\"Radiogroup question\",\"choices\":[\"yes\",\"no\",\"maybe\"]},{\"isRequired\":false,\"enableIf\":true,\"mininumRateDescription\":\"low\",\"name\":\"Rating question\",\"maximumRateDescription\":\"high\",\"type\":\"rating\",\"title\":\"Rating question\"},{\"isRequired\":false,\"enableIf\":true,\"columns\":[{\"text\":\"Strongly Disagree\",\"value\":1},{\"text\":\"Disagree\",\"value\":2},{\"text\":\"Neutral\",\"value\":3},{\"text\":\"Agree\",\"value\":4},{\"text\":\"Strongly Agree\",\"value\":5}],\"name\":\"Please indicate if you agree or disagree with the following statements\",\"type\":\"matrix\",\"title\":\"Please indicate if you agree or disagree with the following statements\",\"rows\":[{\"text\":\"Statement 1\",\"value\":\"Statement 1\"},{\"text\":\"Statement 2\",\"value\":\"Statement 2\"},{\"text\":\"Statement 3\",\"value\":\"Statement 3\"},{\"text\":\"Statement 4\",\"value\":\"Statement 4\"}]}]}";
+            surveysRepository.save(new Survey("First Sample Survey", surveyBody, basicType));
```

Comment:
make sure to not save sample mockups and surveys more than once to repository (similar as in saveSampleProject`)